tokio/src/io/util/mem.rs

```diff
@@ -0,0 +1,178 @@
+//! IO over in-process memory
```

Comment:
```suggestion
//! In-process memory IO types.
```

I feel this is a bit more clear?